src/rng.rs

```diff
@@ -297,13 +214,13 @@ pub trait Rng: RngCore {
     /// use rand::Rng;
     ///
     /// let mut rng = rand::rng();
-    /// println!("{}", rng.gen_ratio(2, 3));
+    /// println!("{}", rng.random_ratio(2, 3));
     /// ```
     ///
     /// [`Bernoulli`]: distr::Bernoulli
     #[inline]
     #[track_caller]
-    fn gen_ratio(&mut self, numerator: u32, denominator: u32) -> bool {
+    fn random_ratio(&mut self, numerator: u32, denominator: u32) -> bool {
```

Comment:
I think this name could be misleading. It sounds as if it generates a "random ratio". But I don't have a better proposal than much longer `random_bool_with_ratio` or the generic proposal, so I guess it's fine.

Reply:
Is code like this confusing? I don't think it's particularly confusing:
```rust
if rng.random_ratio(1, 100) {
   do_something();
}
```

An alternative is that we just remove the method and tell people to use `Bernoulli::from_ratio(n, d).unwrap().sample(&mut rng)`, but I don't see a real reason to remove this little utility fn.